resolwe/flow/tests/__init__.py

```diff
@@ -199,23 +199,22 @@ def run_process(self, process_slug, input_={}, assert_status=Data.STATUS_DONE, r
         for field_schema, fields in iterate_fields(input_, p.input_schema):
             # copy referenced files to upload dir
             if field_schema['type'] == "basic:file:":
-                for app_config in apps.get_app_configs():
-
-                    old_path = os.path.join(app_config.path, 'tests', 'files', fields[field_schema['name']])
-                    if os.path.isfile(old_path):
-                        file_name = os.path.basename(fields[field_schema['name']])
-                        new_path = os.path.join(self.upload_path, file_name)
-                        shutil.copy2(old_path, new_path)
-                        self._upload_files.append(new_path)
-
-                        # since we don't know what uid/gid will be used inside Docker executor,
-                        # we must give others read and write permissions
-                        os.chmod(new_path, 0o666)
-                        fields[field_schema['name']] = {
-                            'file': file_name,
-                            'file_temp': file_name,
-                        }
-                        break
+                file_name = os.path.basename(fields[field_schema['name']])
```

Comment:
Yes, it will work in genialis-base. Only requirement is to set ``self.files_path``in each test class. But this is already required by ``assertFile`` and ``assertJSON``functions, so this change doesn't add any extra complexity to the testing procedure.

Here is example from resolwe-bio: https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/tests/utils.py#L31